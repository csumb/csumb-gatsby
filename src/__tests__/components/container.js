import React from 'react'
import renderer from 'react-test-renderer'
import Container from 'components/common/container'

describe('Container', () => {
  it('renders container correctly', () => {
    const tree = renderer.create(<Container>The children!</Container>).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders container correctly with top padding', () => {
    const tree = renderer
      .create(<Container topPadding>The children!</Container>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
