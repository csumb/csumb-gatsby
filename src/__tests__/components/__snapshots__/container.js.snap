// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders container correctly 1`] = `
<div
  className="css-1qs0ar3"
>
  The children!
</div>
`;

exports[`Container renders container correctly with top padding 1`] = `
<div
  className="css-1tl77yh"
>
  The children!
</div>
`;
